guides/common/modules/proc_chainbooting-ipxe-from-pxelinux.adoc

```diff
@@ -1,53 +1,68 @@
 [id="Chainbooting_iPXE_from_PXELinux_{context}"]
 = Chainbooting iPXE from PXELinux
 
-Use this procedure to set up iPXE to use a built-in driver for network communication or UNDI interface.
-To use HTTP with iPXE, use iPXE build with built-in drivers (`ipxe.lkrn`).
-Universal Network Device Interface (UNDI) is a minimalistic UDP/IP stack that implements TFTP client, however, cannot support other protocols like HTTP (`undionly-ipxe.0`).
-You can choose to either load `ipxe.lkrn` or `undionly-ipxe.0` file depending on the networking hardware capabilities and iPXE driver availability.
+You can set up iPXE to use a built-in driver for network communication (`ipxe.lkrn`) or Universal Network Device Interface (UNDI) (`undionly-ipxe.0`).
+You can choose to load either file depending on the networking hardware capabilities and iPXE driver availability.
 
-.Chainbooting iPXE directly or with UNDI workflow
-. Host powers on
-. PXE driver retrieves the network credentials using DHCP
-. PXE driver retrieves the PXELinux firmware `pxelinux.0` using TFTP
-. PXELinux searches for the configuration file on the TFTP server
-. PXELinux chainloads iPXE `ipxe.lkrn` or `undionly-ipxe.0`
-. iPXE retrieves the network credentials, including an HTTP URL, using DHCP again
-. iPXE chainloads the iPXE template from the template {SmartProxy}
-. iPXE loads the kernel and initial RAM disk of the installer
+UNDI is a minimalistic UDP/IP stack that implements TFTP client, however, cannot support other protocols like HTTP.
+To use HTTP with iPXE, use the iPXE build with built-in drivers (`ipxe.lkrn`).
+
+.Chainbooting iPXE workflow
```

Comment:
I find these repeating dot headers very hard to read because you have 4 headers that look pretty much the same. Please consider changing ".Chainbooting iPXE workflow" to a lead-in sentence like "Chainbooting iPXE has the following workflow:" 

Also, `snip_configuring-templates-provisioning.adoc` with a dot header just looks weird. I realize you might not want to change the snippet in this PR. However, please consider removing the header in the future. IMO, it looks better as a simple paragraph or maybe a note.

I think it's better to reserve dot header formatting mainly for "Prerequisites", "Procedure", and "Additional resources".